packages/build_cmd/lib/repo_finder.dart

```diff
@@ -0,0 +1,103 @@
+import 'dart:io' as io;
+import 'package:args/args.dart';
+import 'package:args/command_runner.dart';
+import 'package:path/path.dart' as path;
+import 'package:io/ansi.dart';
+
+/// Mixin to help [Command] subclasses locate both `google/mediapipe` and
+/// the root of `google/flutter-mediapipe` (this repository).
```

Comment:
Is this intended to be run outside of the context of google/flutter-mediapipe? I wonder if we don't already know the root of this repository, why go looking for it?

Reply:
It's so the command can be run from different locations within the repository. Without this, the `dart ...` command can only be run from whatever path this assumes.

Reply:
A note on the `Outdated` flag - this code has not changed, only moved directories. The above conversation still applies as-is.